modules/insights-operator-configuring.adoc

```diff
@@ -19,30 +19,95 @@ The `ConfigMap` object does not exist by default, so an {product-title} cluster
 //====
 
 .ConfigMap object configuration structure
-This example of an *insights-config* `ConfigMap` object (`config.yaml` configuration) shows configuration options using standard YAML formatting.
+This example of an *insights-config* `ConfigMap` object (`config.yaml` configuration) shows configuration options by using standard YAML formatting.
 
 image::insights-operator-configmap-example.png[Example of Insights Operator ConfigMap object]
 
 .Configurable attributes and default values
-The table below describes the available configuration attributes:
+The following table describes the available configuration attributes:
 
 [NOTE]
 ====
 The *insights-config* `ConfigMap` object follows standard YAML formatting, wherein child values are below the parent attribute and indented two spaces. For the *Obfuscation* attribute, enter values as bulleted children of the parent attribute.
 ====
 
+
 .Insights Operator configurable attributes
 [options="header"]
+[cols=".^2l,.^3a,.^1a,.^1a",options="header"]
```

Comment:
The description table renders really small. Could you expand it? maybe something else could be reduced if there's a size limitation? 

Then again, I'm not exactly sure how this will render on docs.rh

Reply:
Thanks for the review, Steve. I have Value type and Default value columns at the lowest level. The customer portal has a better experience where a customer can expand the table real estate: 

<img width="961" height="258" alt="Screenshot From 2025-08-29 16-19-14" src="https://github.com/user-attachments/assets/6839ce93-93c4-4b9e-9adc-4f333d0f375a" />